.github/workflows/ci.yml

```diff
@@ -90,4 +90,4 @@ jobs:
     with:
       script: .github/scripts/run_unit-tests -Dtest=QTest -Dquidem.split=${{ matrix.split }} -fae
       key: QTest-${{ matrix.split }}
-      jdk: 17
+      jdk: 21
```

Comment:
Should we leave it at the officially "supported" jdk 17? (instead of jdk 21, which still has some problems)

Reply:
Fair question! @gian and I were talking and thought bumping to 21 for qtest now may be a good way to grease the wheels on efforts for gaining confidence in 21, figuring out what is making some of the unit tests fail, and trying to get to full 21 support asap after the druid 24 freeze next week.

Reply:
okay, thanks 👍